### DISCUSSION TITLE
__md_get is not defined when using the privacy plugin

### DISCUSSION THREAD

**Discussion by provDavidLynn** [squidfunk/mkdocs-material#6688] [Q&A] [answered]
Recently added the privacy plugin to our mkdocs which causes an error in the browser that __md_get is not defined.
Is there some setting that I'm missing when using this plugin?

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @provDavidLynn,
since you've closed the discussion I'm assuming you already solved the issue. 
I didn't investigate it too deeply, but just enabling the privacy plugin doesn't remove the `__md_get` function from the HTML.
So perhaps:
- some specific configuration removes it
- your JavaScript runs before it was defined in the HTML source
- some external hook removed it

https://github.com/squidfunk/mkdocs-material/blob/9d52e36212a6d1c32308a5e81084b81eb6f075f2/src/templates/base.html#L153-L154
https://github.com/squidfunk/mkdocs-material/blob/9d52e36212a6d1c32308a5e81084b81eb6f075f2/src/templates/partials/javascripts/base.html#L35-L38

Without a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) I'm just guessing of course.
